public/app/plugins/datasource/prometheus/specs/completer.test.ts

```diff
@@ -5,6 +5,7 @@ import { DataSourceInstanceSettings } from '@grafana/ui';
 import { PromOptions } from '../types';
 import { TemplateSrv } from 'app/features/templating/template_srv';
 import { TimeSrv } from 'app/features/dashboard/services/TimeSrv';
+import { IQService } from 'angular';
```

Comment:
What is this?

Reply:
Typings for q$

Reply:
Haha okay. I thought there was some service for https://en.wikipedia.org/wiki/Intelligence_quotient :laughing: